### DISCUSSION TITLE
Die 7510 ist gefreetzt

### DISCUSSION THREAD

**Discussion by freetz-ng-mod** [Freetz-NG/freetz-ng#520] [Allgemein] [answered]
![freetz_7510](https://user-images.githubusercontent.com/79326105/164011000-2221216a-1112-4d2b-9ee9-3dd61ed82c75.png)
![freetz7510_2](https://user-images.githubusercontent.com/79326105/164011738-efe5b890-c107-4a59-b503-c0b8355f37bd.png)

 -map 0x40000000 -ram 512

**Comment by fda77** [comment]
Thx, nach der run_clock zu urteilen gab es wohl keinerlei Problem
https://github.com/Freetz-NG/freetz-ng/commit/accd7e58e6c3d178d286a18b3df77a7061df9a5b
https://github.com/Freetz-NG/freetz-ng/commit/77e6d8a5a2c4ad59a66119cd5c3b20aaaa46d8ff

**Comment by freetz-ng-mod** [comment, marked as answer]
Genau, habe sie ausgepackt. FRITZ.Box_7510-07.30-recover.exe durchgeführt log gespeichert für hippie und mir die Informationen herausgesucht, die ich gebraucht habe. Und gepusht ;-)

**Comment by pwFoo** [comment]
Hi,
das sind die erforderlichen Optionen um die 7510 zu flashen ?
```
-map 0x40000000 -ram 512
```

Leider kein Effekt bei mir (ist aber wohl auch per commit eingeflossen bereits...). Box ist weiterhin nicht bootfähig. Habe auch alles gelöscht und das image sauber neu gebaut. Ist beim git clone was zu beachten oder beim make? 

Build war lediglich Box ausgewählt. Also wirklich minimal...
